README.md

```diff
@@ -368,5 +368,52 @@ The rule looks for an HTTP header named `X-CRS-Test` and writes its value to the
 
 You can configure the name of the HTTP header by setting the `logmarkerheadername` option in the configuration to a custom value (the value is case insensitive).
 
+## Library usage
+
+`go-ftw` can be used as a library also. Just include it in your project:
+```sh
+go get github.com/coreruleset/go-ftw
+```
+```go
```

Comment:
I'd include the imports here as well, to make it clear which calls refer to go-ftw.

Reply:
Added imports.